talent/job-descriptions/devrel-sf.md

```diff
@@ -0,0 +1,26 @@
+Developer Advocate / Tech Writer
```

Comment:
Are we going to keep both titles or keep one? If we have to choose I would prefer to keep "Developer Advocate".

Reply:
I don't really care much about the title as long as it's clear that there's gonna be lots of writing involved.